src/components/Disputes/DisputeInfo.js

```diff
@@ -62,32 +63,62 @@ const DisputeInfo = React.memo(function({
           <Loading border={false} />
         ) : (
           <>
-            {isFinalRulingEnsured && (
-              <Row>
-                <Field
-                  label="Final jury outcome"
-                  value={
-                    <DisputeOutcomeText
-                      outcome={
-                        lastRound.appeal?.appealedRuling ||
-                        lastRound.vote?.winningOutcome
-                      }
-                      phase={dispute.phase}
-                      disputeEnded={isFinalRulingEnsured}
-                    />
-                  }
-                />
-                <Field
-                  label="Round number"
-                  value={<DisputeRoundPill roundId={dispute.lastRoundId} />}
-                />
-              </Row>
-            )}
             <Row>
-              <Field label="Description" value={description} />
-              {creatorAddress && (
-                <Field label="Plaintiff" value={creatorAddress} />
-              )}
+              {(() => {
+                if (isFinalRulingEnsured) {
+                  return (
+                    <>
+                      <Field
+                        label="Final jury outcome"
+                        value={
+                          <DisputeOutcomeText
+                            outcome={
+                              lastRound.appeal?.appealedRuling ||
+                              lastRound.vote?.winningOutcome
+                            }
+                            phase={dispute.phase}
+                            disputeEnded={isFinalRulingEnsured}
+                          />
+                        }
+                      />
+                      <Field
+                        label="Round number"
+                        value={
+                          <DisputeRoundPill roundId={dispute.lastRoundId} />
+                        }
+                      />
+                    </>
+                  )
+                }
+                return (
+                  <>
+                    <Field label="Description" value={description} />
+                    <div />
```

Comment:
Is this empty div to force a flex layout or for spacing? It seems a bit odd just hanging out here.

Reply:
yeah that is because [here](https://github.com/aragon/court-dashboard/pull/93/files/49ff56e7fe098d5d51cb624566a9e7e1e155b4e3#diff-e447e120bfda3c5da3d624e3174f5292R248) we are setting always the grid template colums as `1fr 1fr 1fr` and if we only need to show the  first element and the last one i did that trick, i wanted to finish it instead of waste time trying to do it with css, but @bpierre was notified by me about that and will check how we can do it

Reply:
Cool! Might just be worth some comments in the future :).